thermo_nerf/thermal_nerf/calculate_threshold.py

```diff
@@ -20,7 +22,10 @@ def calculate_threshold(data: Path):
         data_dir = data
     threshold_list = []
     for frame in meta["frames"]:
-        filepath = Path(frame["thermal_file_path"])
+        if model_type == ModelType.NERFACTO:
```

Comment:
You can avoid if/else by giving a default value and changing it "if" model is needs to only. It makes the code easier to read than if/else

Start by giving it the value "thermal_file_path" to a key and change it if nerfacto